app/controlplane/pkg/biz/workflowcontract.go

```diff
@@ -370,6 +383,11 @@ func (uc *WorkflowContractUseCase) findAndValidatePolicy(att *schemav1.PolicyAtt
 	// [chainloop://][provider:][org_name/]name
 	if loader.IsProviderScheme(att.GetRef()) {
 		pr := loader.ProviderParts(att.GetRef())
+		// Validate attachment
+		if err := uc.ValidatePolicyAttachment(pr.Provider, att, token); err != nil {
```

Comment:
should we handle the case where the API doesn't exist in the provider backend? Maybe handling 404 would be enough?

Reply:
Right, we can add it and no-op in that case.

Reply:
Done in the provider implementation